tests/src/JIT/Performance/CodeQuality/BenchI/XposMatrix/XposMatrix.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Should we put this directory at the top level? (Performance).  CoreRT may also use these.